obspy/io/shapefile/core.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future.utils import native_str
+
+from obspy import Catalog
+from obspy.core.event import Origin, Magnitude
+from obspy.core.inventory import Inventory
+
+
+try:
```

Comment:
I just thought I'd point out more clearly that this needs an additional module to be installed which is not a hard dependency of obspy and thus likely not yet installed. I think otherwise it's really surprising to run into an `ImportError` when using a package that otherwise handles all dependencies via the packaging system.

How about I reraise with the original message and append some more information to it?